### PR TITLE
feat: implement v3 style option function

### PR DESCRIPTION
**Pull request by wolf-joe**
with concurrent safe

### CODE REVIEW

File: pkg/options/options.go [hunk 1]
```diff
@@ -9,6 +9,16 @@ import (
...
 
+var (
+	generateUniqueValues = false
+	ignoreInterface      = false
+	randomStringLen      = 25
+	lang                 = &interfaces.LangENG
+	randomMaxSize        = 100
+	randomMinSize        = 0
+	iBoundary            = &interfaces.DefaultIntBoundary
+)
```
Comment (bxcodec) [inline pkg/options/options.go:20]: do we really need to support this? It will be unsafe for concurrent use.

Comment (wolf-joe) [reply pkg/options/options.go:20]: We can use the atomic operation to support concurrent use cases if it's needed. But in my opinion, a global setting is always valuable, especially when it already supports in the old version.

Comment (bxcodec) [reply pkg/options/options.go:20]: hmmm, I'm a bit reluctant about this.
But, if we want this feature, let's ensure it's safe for concurrent use. Can we have that?

**Review by bxcodec — APPROVED** [review]
LGTM

**Pull request merged**